File: get-the-middle-character/Kata.java
class Kata {
    public static String getMiddle(String word) {
        if (word.length() % 2 == 0) {
            int half = word.length() / 2;
            return word.substring(half - 1, half + 1);
        } else {
            int half = word.length() / 2;
            return word.substring(half, half + 1);
        }
    }
}

File: get-the-middle-character/kata.js
function getMiddle(s) {
  if (s.length % 2 == 0) {
    const half = s.length / 2;
    return s.substring(half - 1, half + 1);
  } else {
    const half = s.length / 2;
    return s.substring(half, half + 1);
  }
}